tardis/plasma/properties/nlte_rate_equation_solver.py

```diff
@@ -94,17 +105,16 @@ def calculate(
         )  # dropping the n_e index, as rate_matrix_index's first index is (atomic_numbers, "n_e")
 
         index = rate_matrix_index.droplevel("level_number").drop("n_e")
-        ion_number_density_nlte = pd.DataFrame(
-            0.0, index=index, columns=phi.columns
-        )
-        electron_densities_nlte = pd.Series(0.0, index=phi.columns)
+        ion_number_density = pd.DataFrame(0.0, index=index, columns=phi.columns)
```

Comment:
This is minor, but for easier code comparison, it is helpful to use the formatter such as black.